department-of-veterans-affairs/caseflow
# Justin/APPEALS-50741

Resolves [Disable check on Correspondence Details to not allow any users](https://jira.devops.va.gov/browse/APPEALS-50741)

# Description
Edited routing and permissions to allow all Caseflow users to access correspondence details instead of just users part of the inbound ops team organization. Created new tests to verify this new functionality.

## Acceptance Criteria
- [ ] In the Correspondence Details controller override verify_correspondence_access to allow any Caseflow user access to the Correspondence Details Page

## Testing Plan

Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
1. Log in as an inbound ops team member with cases, or as an inbound ops team admin user with access to all cases. 
2. Go to the review package page for a correspondence case and copy the URL for the page. You will need this URL for a later step.
3. Next, log in as a user that is not a member of the inbound ops team. Paste in the URL you copied in the previous step and attempt to navigate to the page. _User will be routed to the unauthorized page_
4. Now, paste your URL into the address bar again. This time, remove the /review_package part of the URL first. _User will now be able to see the Correspondence Details page. Any other page within the correspondence workflow (besides the details page) should still be disabled for the user._
5. Please note: All users should be able to access the correspondence details page for a specific correspondence - found at /queue/correspondece/{correspondence UUID} - however, only Inbound Ops Team users should be able to access the rest of the correspondence workflow.
